linux/helper_centos.sh

```diff
@@ -1,19 +1,11 @@
 # Adds the NR repo
 add_repo() {
-    yum update -y
-    yum install -y wget
-
+    cp newrelic-infra-centos.repo /etc/yum.repos.d/newrelic-infra.repo
     if [ "$STAGING_REPO" = "true" ]; then
-        repo="http://nr-downloads-ohai-staging.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/yum/el/8/x86_64/newrelic-infra.repo"
-    else
-        repo="http://nr-downloads-main.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/yum/el/8/x86_64/newrelic-infra.repo"
+      rs='s|nr-downloads-main|nr-downloads-ohai-staging|'
+      sed -i "$rs" /etc/yum.repos.d/newrelic-infra.repo
     fi
```

Comment:
Rather than doing copy-then-sed, I'd rather have `.repo` files specifically for staging. E.g.:
- `newrelic-infra-centos.repo`
- `newrelic-infra-centos-statging.repo`
- etc.